docs/content/querying/timeseriesquery.md

```diff
@@ -13,6 +13,7 @@ An example timeseries query object is shown below:
   "queryType": "timeseries",
   "dataSource": "sample_datasource",
   "granularity": "day",
+  "descending": "true",
```

Comment:
this flag looks to be on the BaseQuery object, which means this code should work for all query types no?